app/concepts/phytosanitary/risk.rb

```diff
@@ -15,7 +15,7 @@ def initialize(risk_level)
     end
 
     def self.risks_of(variant)
-      return [Unknown] unless maaid = variant.france_maaid
+      return [Unknown] unless maaid = variant.nil? ? nil : variant.france_maaid
```

Comment:
I think that the problem is more in: `app/views/backend/interventions/_parameter_form.html.haml`:25 the list of products contains `nil` values.